src/utils/api/api.ts

```diff
@@ -171,14 +172,17 @@ export const removeReadonlyFields = <T extends z.ZodRawShape, TUnwrap extends (k
   shape: T,
   unwrap?: TUnwrap
 ) => {
-  const entries = Object.entries(shape).flatMap(([k, v]) => {
-    if (v instanceof z.ZodBranded && v.description === READONLY_TAG) {
```

Comment:
This was really confusing since the test passed properly but whenever running the library within vue/runjs this seemed to work differently.
React yielded `v instanceof z.ZodBranded` to be `true`
But Vue and RunJS would just say `false`
This just threw me off and made this quite hard to debug.